PhIPSeq_external/Analysis_allergens_and_IEDB/predict_age.py

```diff
@@ -0,0 +1,121 @@
+import os
+import string
+import seaborn as sns
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+from scipy.stats import fisher_exact
+from sklearn.metrics import r2_score
+from statsmodels.stats.multitest import fdrcorrection
+
+from PhIPSeq_external.Analysis_allergens_and_IEDB.config import base_path, out_path
+from PhIPSeq_external.Analysis_allergens_and_IEDB.prediction_helpers import get_oligos, configure_x_y, \
+    tune_hyper_parameters, predict_with_cross_validation, plot_shap, read_metadata
+
+
+def compute_age_correlations(x, y, overwrite=True):
+    csv_out_path = os.path.join(out_path, 'age_correlations_data.csv')
+    if overwrite or not os.path.exists(csv_out_path):
+        os.makedirs(out_path, exist_ok=True)
+        bottom_quintile = y.quantile(0.2)
+        top_quintile = y.quantile(0.8)
+        bottom_label = f"Percent of youngest quintile (<{int(bottom_quintile)}\nin whom a peptide is significantly bound"
+        top_label = f"Percent of oldest quintile (>{int(top_quintile)}\nin whom a peptide is significantly bound"
+        age_group = y.apply(
+            lambda v: bottom_label if v < bottom_quintile else (top_label if v > top_quintile else np.nan))
+        oligos = x.drop(columns=['gender']).gt(1)
```

Comment:
why not gt(0) ?
this is log fold.... (in practive we don;t have any, but we could have had)